vignettes/datasets.Rmd

```diff
@@ -26,8 +26,10 @@ To create new datasets, multiple paths exist. In the web application, you can up
 
 You can create a dataset from any `data.frame` you have in your R session with `newDataset`:
 
-```{r, eval=FALSE}
-load("economist.RData")
+This package provides one as `economist` which loads a `data.frame` named `df` because old habits die hard.
+
+```{r}
+data("economist", package = "crunch") # named df
```

Comment:
@GitBrianLaw can add that. I thought we did. All of the issues can be resolved by renaming in `1-getting-started.R`

```
economist <- df
save(economist, file="../data/economist.RData")
```